install-components.md

```diff
@@ -1883,153 +1883,64 @@ To install Supply Chain Security Tools - Scan (Scan Controller):
 
-1. If [Supply Chain Security Tools - Store](#install-scst-store) is installed, configure a TLS Certificate `Secret` and Read-Write `ClusterRole` for the [Supply Chain Security Tools - Store](#install-scst-store).
-
-    The TLS Certificate necessary for sending requests to the [Supply Chain Security Tools - Store](#install-scst-store) resides in the `metadata-store` namespace. Create the following secret in the `scan-link-system` by copying the CA Certificate from the `metadata-store` namespace by running:
-
-    ```bash
-    kubectl create namespace scan-link-system
-    kubectl create secret generic metadata-store-ca \
-      -n scan-link-system \
-      --from-file=ca.crt=<(kubectl get secret app-tls-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d)
-    ```
-
-    Create the necessary `ClusterRole` for sending requests to the [Supply Chain Security Tools - Store](#install-scst-store) by running:
-
-    ```console
-    kubectl apply -f - -o yaml << EOF
-    ---
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: ClusterRole
-    metadata:
-      name: metadata-store-read-write
-    rules:
-    - apiGroups: [ "metadata-store/v1" ]
-      resources: [ "all" ]
-      verbs: [ "get", "create", "update" ]
-    EOF
-    ```
-
-1. If [Supply Chain Security Tools - Store](#install-scst-store) is installed, create a `scst-scan-controller-values.yaml` using the following sample as a guide.
-
-    Sample `scst-scan-controller-values.yaml` for Scan Controller:
-
-    ```yaml
-    ---
-    metadataStore.url: https://metadata-store-app.metadata-store.svc.cluster.local:8443
-    metadataStore.caSecret: metadata-store-ca
-    metadataStore.clusterRole: metadata-store-read-write
-    ```
-
-    The following shows how to determine what these values are:
-
-    The `metadataStore.url` value can be determined by:
-
-    ```bash
-    kubectl get service -n metadata-store -o name |
-      grep app |
-      xargs kubectl -n metadata-store get -o jsonpath='{.spec.ports[].name}{"://"}{.metadata.name}{"."}{.metadata.namespace}{".svc.cluster.local:"}{.spec.ports[].port}'
-    ```
-
-    The `metadataStore.caSecret` value is the name of the TLS Certificate `Secret` created above.
-
-    The `metadataStore.clusterRole` value is the name of the `ClusterRole` created above.
-
-4. If [Supply Chain Security Tools - Store](#install-scst-store) is installed, install the package by running (omit the `--values-file` line if installing without [Supply Chain Security Tools - Store](#install-scst-store) already installed):
+1. Install the package by running (omit the `--values-file` line if installing with default configuration):
 
     ```console
     tanzu package install scan-controller \
-      --package-name scanning.apps.tanzu.vmware.com \
-      --version 1.0.0-beta.2 \
+      --package-name scst-scan.apps.tanzu.vmware.com \
+      --version 1.0.0 \
       --namespace tap-install \
       --values-file scst-scan-controller-values.yaml
```

Comment:
Isn't it clear to me ? Do we really need `scst-scan-controller-values.yaml` now ?

Reply:
The users would copy and paste this command, so better suggest it to add a yaml file but leave the command example as simpler as possible.